.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,20 +0,0 @@
----
```

Comment:
I don't see any reason to delete this templates.

Reply:
reverted